Modules/Services/Sources/Models/Block/BlockContent/Content/BlockRelation.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
 public struct BlockRelation: Hashable {
-    public let key: String
```

Comment:
It might be alright to change these variable to a mutating ones due to legacy logic for handling all middleware updates.
Imho for better handling errors and converter issues we need to leave it immutable.

Reply:
What advantage do you see in updating via init? We have a public init, we don't blocking the rewrite, but make rewrite more harder.

Reply:
you can be confident about your model hasn't been updated somewhere in code. And it could be created only in one place -> where we receiving value from middleware